main.py

```diff
@@ -38,10 +38,10 @@ def create_certificate(crds, namespace, secretname, routes):
     Create a certificate request for certmanager based on the IngressRoute
     """
     try:
-        secret = crds.get_namespaced_custom_object(CERT_GROUP, CERT_VERSION, namespace, CERT_PLURAL, secretname)
+        # secret = crds.get_namespaced_custom_object(CERT_GROUP, CERT_VERSION, namespace, CERT_PLURAL, secretname)
```

Comment:
I think if this is commented out it will never hit the exception and thus will always return and not create the secret.

Reply:
I see. That's my mistake for not understanding the purpose of that line and blindly linting. Perhaps instead of assigning an unused variable, you replace it with an "assert" statement. See [this commit](https://github.com/ncsa/traefik-certmanager/pull/2/commits/aeaccb9d140f79ed595ec5af613bf07788baaf67).

https://github.com/ncsa/traefik-certmanager/pull/2/commits/aeaccb9d140f79ed595ec5af613bf07788baaf67#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R41